netsupportpolicy.md

```diff
@@ -0,0 +1,16 @@
+.NET support rules:
+1. ODL does not support .NET versions that are out-of-support
+2. .NET support has been tested prior to publishing a version with that .NET support
+3. removing support for a .NET version is a breaking change (and therefore requires a major version increment)
+4. adding support for a .NET version is not a breaking change
+5. ODL major versions are support for at least 1 year after release (TODO this claim was made in the last meeting, but i can't find in the existing doc)
+

```

Comment:
```suggestion
6. Any time .NET releases a new GA version, we have a GA version of OData that has been tested to work with that version of .NET
7. Any time .NET releases a new beta version, we have a beta or GA version of OData that has been tested to work with that beta version within 3 months
 ```

Reply:
we need to also make sure that we know (for ourselves) that breaking chjanges are the *only* way to do some things

Reply:
maybe we start with **only** .NET LTS releases to start with